applications/trunkstore/src/ts_callflow.erl

```diff
@@ -124,8 +124,9 @@ route_won(#ts_callflow_state{amqp_worker=Worker, kapps_call=Call}=State, RouteWi
 
     lager:info("callflow has received a route win, taking control of the call"),
 
-    {'won', State#ts_callflow_state{callctl_q=kapi_route:control_queue(RouteWin)
-                                   ,kapps_call=kapps_call:from_route_win(RouteWin, Call)
+    NewCall = kapps_call=kapps_call:from_route_win(RouteWin, Call),
```

Comment:
This seems wrong. Is the atom supposed to be `#kapps_call{}` instead? Or just remove it?